### ISSUE TITLE
Feature Request: option to not run scripts after insertion

### ISSUE THREAD

**Issue by dimateos** [YashTotale/terminal-all-in-one#414]
_Hey, thanks for the great extension!_

**Is your feature request related to a problem? Please describe.**
The scripts functionality is very useful, but I feel like the fact that the snippets are always auto run is a limiting factor. When a command is very similar, it's easier to have a single snippet and tune it before runnning, e.g. here I dont want two snippets just to remove "-no":

```json
"terminalAllInOne.scripts": [
    {
        "name": "git: consider unchaged",
        "script": ["git update-index --no-assume-unchanged"],
    },
]
```

**Describe the solution you'd like**
Probably a key added to the json object toggling the behavior:

```json
"terminalAllInOne.scripts": [
    {
        "name": "git: consider unchaged",
        "script": ["git update-index --no-assume-unchanged"],
        "run": false,
    },
]
```

**Additional context**
This also seems useful for scripts you need to complete, e.g. like a final argument etc.
* I would be similar but way more simpler than vscode inputs in tasks: https://stackoverflow.com/questions/45582698/is-it-possible-to-pass-arguments-to-a-task-in-visual-studio-code
* Basically similar to inserting snippets, which is not possible in the terminal

**Comment by marktenney** [comment]
I came to request this exact same feature and was pleased to see this already posted. I've been adding placeholders into my scripts to prevent them from running on insertion, just so I can let the system try to run it and fail, then hit the up arrow to queue up the command again and edit it before running. Commands should just be entered and not run.

**Comment by marktenney** [comment]
@YashTotale Have you given this any thought? It would be great to either have a setting to disable autorun or the ability to set specific scripts to not run automatically.
